requirements-dev.txt

```diff
@@ -0,0 +1,5 @@
+black==19.10b0
+faker
+pytest
+pytest-docker
```

Comment:
We'll probably want to pin these so we don't run into the problem we saw in #327 (if I'm understanding that one correctly).

```suggestion
Faker==6.5.*
pytest==6.2.*
pytest-docker==0.10.*
```

Reply:
Oh yeah, good call there also. Thanks!